Assets/js/popupmodal.js

```diff
@@ -0,0 +1,24 @@
+const popUpModal = () => {
+  document.addEventListener("DOMContentLoaded", (event) => {
+    const okbtn = document.getElementById("cancelBtn");
+    okbtn.addEventListener("click", () => {
+      document.getElementById("myModal").classList.remove("show");
+    })
+
+    // buttons to load the pop up modal
+    document.getElementById("successPopUp").addEventListener("click", () => {
+        document.getElementById("myModal").classList.add("show")
+        document.querySelector(".cmp-custom-modal__content-icon").setAttribute("src", "../../images/success_icon.png");
+        document.querySelector(".cmp-custom-modal__content-title").innerHTML = "Hurrah! Your message sent";
```

Comment:
It's a little unusual to have the javascript write the text for a popup, but it definitely works.

Reply:
@mathrik , You are absolutely right, I have set these text only to demonstrate and once it is added as AEM component then all text and images will be set from cq_dialogue and will required small changes. I have added these text through js to show two purposes for this pop up modal, which can be used in your pages or forms.